Fw/Types/DefaultTypes.hpp

```diff
@@ -0,0 +1,15 @@
+
+#ifndef PLATFORM_INT_TYPE_DEFINED
+typedef int PLATFORM_INT_TYPE;
+#define PRIfwpit "d"
```

Comment:
Shouldn't this be "%d"?

Reply:
No. In C/C++, the PRI* macros are defined without the "%" symbol.